Codecademy/gamut
# feat: add new Skills icons

### Overview



- Added new icons for languages and subjects that we teach
- Organized these icons into a new Skills section in storybook



Note: there were a few icons that were already in gamut but just needed to be moved into the Skills section and the RubyIcon needed to be updated and moved. All details are in the [Figma](https://www.figma.com/design/h5Opgw8Inv1KSSsek0aawR/%F0%9F%8C%A0-Skill-XP?node-id=389-14536&t=p2rIbAzaLaBjzFSM-0). 

### PR Checklist


#### Testing Instructions



1. Visit the storybook preview and navigate to the Regular Icons section.
2. Ensure that you see a new Skills section at the bottom and that all the icons are there and look good.
3. You can also checkout the mono repo PR to see it used in the new skills section on the dashboard. 

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/6535)  | [Portal Env](https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-6535)   |
